### DISCUSSION TITLE
Blank page after upgrade to v 2.3.9

### DISCUSSION THREAD

**Discussion by mott2060** [Leantime/leantime#1412] [General] [answered]
Hi all,
I´ve just tried to update from version 2.3.6 to version 2.3.9 by uploading all new files via FTP to my server - like I always did in the past with any new version. However, it seems like something went different this time since I only can see a blank page anymore.
Log files says:
[12-Feb-2023 09:26:45 Europe/Vienna] PHP Fatal error:  Uncaught Error: Call to undefined method leantime\core\appSettings::getRequestURI() in /html/leantime/public/index.php:17
Stack trace:
#0 {main}
  thrown in /html/leantime/public/index.php on line 17

Any suggestions what I can do now?

Thank you!

**Comment by TareqK** [comment, marked as answer]
See the patch #1411 - you can manually patch your appsettings file under conf, and we'll try and get this fixed ASAP

**Reply by mott2060** [reply]
Thanks! That fix worked for me!
